src/Aquifer.Jobs/host.json

```diff
@@ -1,60 +1,53 @@
 {
     "version": "2.0",
-
     "logging": {
         "logLevel": {
             "default": "Information",
-            "Azure.Core": "Error" // suppress noisy SDK warnings
+            "Azure.Core": "Error"
+            // suppress noisy SDK warnings
```

Comment:
This comment applies to the preceding line and should be moved above it if the formatter doesn't like same line comments.